adam-core/src/test/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDSuite.scala

```diff
@@ -315,8 +377,9 @@ class AlignmentRecordRDDSuite extends ADAMFunSuite {
     val rRdd = sc.loadAlignments(inputPath)
     rRdd.rdd.cache()
     rRdd.saveAsSam("%s/%s".format(tempFile.toAbsolutePath.toString, filename),
-      asSam = asSam,
+      asType = if (asSam) Some(SAMFormat.SAM) else Some(SAMFormat.BAM),
       asSingleFile = true)
+    println("Saved file to %s/%s".format(tempFile.toAbsolutePath.toString, filename))
```

Comment:
Dur, had that in for debugging. I'll remove this, then squash and rebase. Thanks for catching this @heuermh !